torch/_inductor/decomposition.py

```diff
@@ -22,13 +22,52 @@
         aten.linalg_vector_norm,
         aten.std,
         aten.std_mean,
-        aten._to_copy,
         aten.triu_indices,
         aten.tril_indices,
         aten.sqrt_,
         aten.lcm,
         aten.clamp_min_,
         aten.sin_,
+        aten.addcdiv,
+        aten.addr,
+        aten.binary_cross_entropy,
+        aten.bitwise_and_,
+        aten.bitwise_or_,
+        aten.binary_cross_entropy_backward,
+        aten.celu,
+        aten.diag_embed,
+        aten.diagonal,
+        aten.frac,
+        aten.hardshrink,
+        aten.heaviside,
+        aten.huber_loss,
+        aten.isneginf,
+        aten.isposinf,
+        aten.logical_xor,
```

Comment:
imo better to have lowering for this (together with other logical_* ops)

Reply:
Sure, will add those

Reply:
other `logical_*` already exist in lowering, so `logical_xor` better fits with existing `logical_and` and friends,  and `@register_pointwise` has `convert_to_bool` arg which is pretty much what decomp is doing.